samples/caf/README.rst

```diff
@@ -50,9 +50,9 @@ User interface
 Button 1:
   Changes the LED effect displayed on the LED to the next defined LED effect:
 
-  * The first default LED effect toggles the blinking of the LED, with the period set to 200 ms.
+  * The first default LED effect toggles the blinking of the LED, with the period set to 400 ms.
   * The second default LED effect toggles the breathing of the LED, with the period set to 400 ms.
-  * The third default LED effect toggles a looped effect where blinks with the period set to 200 ms and then turns off for 1000 ms.
+  * The third default LED effect toggles a looped effect where blinks with the period set to 400 ms and then turns off for 1000 ms.
 
 Button 2:
    Switches the active LED: turns off the currently active LED and turns on the next available LED.
```

Comment:
Does this line need to change as well?

Reply:
Yes! I missed that, thanks for spotting...

Reply:
@wiba-nordic  What do you think about:
```
Switches the active LED: leaves the currently active LED with displayed LED effect and turns on the next available LED.

Reply:
I'd maybe go with:
`Switches the active LED to the next available LED. 
The previously active LED keeps its LED effect, and the new active LED turns on.`